pnglibconf.h

```diff
@@ -0,0 +1,216 @@
+/* libpng 1.6.18 STANDARD API DEFINITION */
+
+/* pnglibconf.h - library build configuration */
+
+/* Libpng version 1.6.18 - July 23, 2015 */
+
+/* Copyright (c) 1998-2014 Glenn Randers-Pehrson */
+
+/* This code is released under the libpng license. */
+/* For conditions of distribution and use, see the disclaimer */
+/* and license in png.h */
+
+/* pnglibconf.h */
+/* Machine generated file: DO NOT EDIT */
+/* Derived from: scripts/pnglibconf.dfa */
```

Comment:
I removed a project that contained no files at all, that existed just to copy from a .h.prebuilt to this file location.  As I think about that further, that's probably a bad idea ... I'll go ahead and restore that project, so that if we pull in future changes that file can update properly.